### PR TITLE
fix: typos in the italian sentences

### PR DESCRIPTION
**Pull request by Marlowess**

### CODE REVIEW

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰


✅ All configuration files are in line with the master branch.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #74 (id: 866034fa-bf70-4a7d-a61b-f4754c2f0a85)
- Last verified commit: #d5f033ccf085ef25ccf93dc02f99af698bef0e09
- Time of check: 25/05/2020, 13:16:02 UTC
- The following protected files have been checked for changes: app/build.gradle, ci/danger/commitlint/rules.ts, build.gradle, ci/changelog_config.tpl, fastlane/Appfile, Gemfile, ci/danger/ktlint.ts, gradle.properties, .editorconfig, fastlane/Fastfile, ci/danger/commitlint/index.ts, ci/danger/danger.ts, gradlew, scheduler_config.json, settings.gradle, package.json, template.properties, publiccode.yml, Gemfile.lock, app/proguard-rules.pro, yarn.lock, gradlew.bat, .gitmodules, dangerfile.ts.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -148,7 +148,7 @@
     <string name="close_contact_warning_title">Rilevato un contatto ad alto rischio</string>
     <string name="close_contact_warning_title_short">Contatto ad alto rischio</string>
     <string name="close_contact_warning_subtitle">Immuni ha rilevato che due giorni fa hai avuto l’ultimo contatto con un utente positivo al COVID-19.</string>
-    <string name="close_contact_suggested_action"><b>Contatta immediatamente il Dipartimento di Prevenzione della tua ASL,</b> se non lo hai già fatto.</string>
```
Comment (andybergon) [inline app/src/main/res/values/strings.xml:151]: If you Google for "Dipartimento di Prevenzione" the `P` is almost always capitalized.

Comment (Marlowess) [reply app/src/main/res/values/strings.xml]: Updated.

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰
<ul>
<li>The following files have been modified: app/build.gradle.</li>
</ul>

⚠️ Some configuration files don't match the master branch.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #174 (id: 8294b3bc-e97b-4948-8700-6d355b7752b5)
- Last verified commit: #76ce7708e9a699a7a8fb85f3dbf7f0d4215390da
- Time of check: 26/05/2020, 07:30:57 UTC
- The following protected files have been checked for changes: scheduler_config.json, gradlew.bat, dangerfile.ts, build.gradle, ci/danger/commitlint/index.ts, gradle.properties, package.json, ci/danger/ktlint.ts, gradlew, Gemfile, yarn.lock, fastlane/Appfile, settings.gradle, app/proguard-rules.pro, template.properties, fastlane/Fastfile, ci/danger/commitlint/rules.ts, .editorconfig, .gitmodules, app/build.gradle, Gemfile.lock, publiccode.yml, ci/danger/danger.ts, ci/changelog_config.tpl.

**Review by VaiTon — APPROVED** [review]

**Pull request closed without merging**